srcpkgs/lcalc/template

```diff
@@ -0,0 +1,76 @@
+# Template file for 'lcalc'
+pkgname=lcalc
+version=1.23
+revision=1
+build_wrksrc="src"
+build_style="gnu-makefile"
+makedepends="gmp-devel pari-devel"
+short_desc="Michael Rubinstein's L-function calculator"
+maintainer="Gonzalo Tornaría <tornaria@cmat.edu.uy>"
+license="GPL-2.0-or-later"
+homepage="https://mirrors.mit.edu/sage/spkg/upstream/lcalc/"
+distfiles="https://mirrors.mit.edu/sage/spkg/upstream/lcalc/lcalc-${version}.tar.bz2"
+checksum=83d4253ec2f38553b21190d6d0c6b71bab7ea14717f6dde5bd18f60775d3cdb0
+
+# get patches from sagemath
+_patchurl="https://git.sagemath.org/sage.git/plain/build/pkgs/lcalc/patches"
+_patchtag="?h=9.3.rc2"
+_patches="
+ Lcommon.h.patch
+ Lvalue.h.patch
+ Makefile.patch
+ clang.patch
+ lcalc-1.23_default_parameters_1.patch
+ lcalc-1.23_default_parameters_2.patch
+ pari-2.7.patch
+ pari-mem.patch
+ pari_include.patch
+ time.h.patch
+ using_namespace_std.patch
+"
+checksum+="
+ bd39a43b24a332e1826e667f85ca2ce2ca14749d56f60069cd886c96f55e9e3c
+ d8c660f68f33b6eeb4b801d4028faaaaccbfff5797efe654bb6367e0a687542d
+ dadf6fc6d481595cb041f8b0dd92bc2e2bc390cbb3e0dd36b7fe63b74942db26
+ 342cde1c9effda48597bfee439acb37c91458a100580c97c84da6a142d2cc5b6
+ 55c6c9f969593f24c75a85e7a150b876671356aa68ceb8e77c563a645897643e
+ 906ef33de15bb0d94883b7797d8cb3c889b9f59bceebeb171e65f1546faf1c37
+ 437e3870342ef4cb5749398ead9381504403e7e1f99efadab95f4a607678b400
+ 6471f2f30497486e289fc8977cbb89c495cf0d5ff58af2b1f479a497a28165e6
+ dde1f80b0ca42fb43917c24e6016c19e4a8d28de9c50cd0a523a318866bdc0eb
+ 941d4ac025f5d6f0142fb785b27f7a221729466dc83d45fd206cce102c012eaf
+ 045f23ea9131b8039001c8f13d22152f7b3c2e91adcf8c555354eaf39e7a7b48
+"
+for patch in $_patches ; do
+	distfiles+=" ${_patchurl}/${patch}${_patchtag}>${patch}"
+done
+skip_extraction="$_patches"
+
+post_extract() {
+	for patch in $_patches ; do
+		patch -p1 < "$XBPS_SRCDISTDIR/${pkgname}-${version}/${patch}"
+	done
+}
+
+pre_build() {
+	CXXFLAGS="-fPIC"
+}
+
+pre_install() {
+	mkdir -p ${DESTDIR}/usr/{bin,lib,include}
+	make_install_args+=" INSTALL_DIR=${DESTDIR}/usr"
+}
+
+do_check() {
+	echo "Testing lcalc ..."
+	sh "${FILESDIR}"/test > test.log
+	diff "${FILESDIR}"/test.out test.log && echo PASS
+}
+
+lcalc-devel_package() {
+	short_desc+=" - development files"
+	pkg_install() {
+		vmove "usr/lib"
```

Comment:
Does sage use the binary or the library? If it's the library, I think it should go in the normal package...

Reply:
I think both. You are right, the `.so` should probably go in the main package. For some reason the binary is compiled static so it doesn't need the `.so` to be around.